### PR TITLE
ci-check: use temporary repo's

### PR DESCRIPTION
**Pull request by naveen521kk**

### CODE REVIEW

**Comment by naveen521kk** [comment]
@lazka does this look good?

File: .ci/ci-check.py [hunk 1]
```diff
@@ -116,9 +116,9 @@ def install_package(pkg: typing.Union[typing.List[str], Path]) -> typing.Generat
         ret = subprocess.run(command)
     if ret.returncode != 0:
         logger.error(
-            "%s failed to run. Returned return code: %s", command, ret.returncode
+            "[WARNING] %s failed to run. Returned return code: %s", command, ret.returncode
```
Comment (jeremyd2019) [inline .ci/ci-check.py:119]: I think you want https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message

It says file/line/col are optional, I've used `::error ::<message>` don't know if the space after error is required or not

File: .ci/ci-check.py [hunk 2]
```diff
@@ -116,9 +116,9 @@ def install_package(pkg: typing.Union[typing.List[str], Path]) -> typing.Generat
         ret = subprocess.run(command)
     if ret.returncode != 0:
         logger.error(
-            "[WARNING] %s failed to run. Returned return code: %s", command, ret.returncode
+            "::error :: %s failed to run. Returned return code: %s", command, ret.returncode
```
Comment (jeremyd2019) [inline .ci/ci-check.py:119]: these should have been `::warning::` I think

Comment (naveen521kk) [reply .ci/ci-check.py:119]: error looks right here

**Pull request merged**